src/matrix/lkj.jl

```diff
@@ -159,7 +159,7 @@ function _marginal(lkj::LKJ)
     d = lkj.d
     η = lkj.η
     α = η + 0.5d - 1
-    LocationScale(-1, 2, Beta(α, α))
+    return 2 * Beta(α, α) - 1
```

Comment:
Do you mind following the convention, that we "tend to space operators more tightly if they get applied before other nearby operators"
```suggestion
    return 2*Beta(α, α) - 1
```